spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -2488,7 +2487,7 @@ properly closed by the time the end of the page is
 reachedfootnote:[If the responsibility was left to the consumer tag, this could lead to resource leaks (e.g. connection left open, memory space for buffers) until garbage collection is activated. This is because a consumer tag might not close the _Reader_, or because the page author might remove the consumer tag while leaving inadvertantly the <c:import> tag in the page.]. Because of this requirement, the Jakarta Standard Tag Library defines
 the exported _Reader_ as having nested visibility: it may not currently
 be accessed after the end-tag for the `<c:import>`
-actionfootnote:[This restriction could eventually be lifted when the JSP spec supports the notion of page events that actions could register to. On a _pageExit_ event, an <c:import> tag would then simply release its resources if it had not already been done, removing the requirement for nested visibility.]. Implementations that use JSP 1.2
+actionfootnote:[This restriction could eventually be lifted when the Jakarta Server Pages spec supports the notion of page events that actions could register to. On a _pageExit_ event, an <c:import> tag would then simply release its resources if it had not already been done, removing the requirement for nested visibility.]. Implementations that use JSP 1.2
```

Comment:
How should we handle the "JSP 1.2" occurrence here?

Reply:
I think we simply don't need to qualify it as the "JSP 1.2 tag-extension API". Just "tag-extension API" should do.